docs/prysm-usage/get-more-peers.md

```diff
@@ -0,0 +1,59 @@
+
+> NOTICE: This section contains advanced network configurations and is optional.
+
+ETH2 leverages [libp2p](../how-prysm-works/p2p-networking), a framework and suite of protocols for building peer-to-peer network applications.  When a beacon-chain node first starts up, it does two things to start communicating with other participants: 
+- It begins listening for new incoming p2p connections
+- It starts a [discovery](https://github.com/ethereum/devp2p/wiki/Discovery-Overview) process to find and connect to new peers
+
+#### Home Networks & Routers
+
+Many participants on the ETH2 network operate their nodes on a home network. Home networks typically have a router that provides a logical boundary between your private home network, and the public internet.  While this is good for keeping bad traffic out of your network, it presents a challenge for communicating with other nodes who are also on home networks.
+
+#### Virtual Public Cloud (VPC) Networks
+
+Other participants on the ETH2 network operate their nodes on a virtual public cloud (VPC) instance.  This is basically a computer running in a datacenter that quite often is directly connected to the public internet.
+
+#### Incoming p2p connection prerequisites
+
+In order for other participants on the ETH2 network to establish incoming p2p connections with your beacon-chain node, a number of conditions must be met:
+1) Your public IP address must be known.
+2) The protocol (TCP/UDP) and port number (0-65535) on which your beacon-chain node is listening must be known (Default - TCP/13000).
+3) All routers & firewalls must be configured to allow incoming traffic on that protocol/port combination.
+
+#### Private & Public IP Addresses
+
+Computers on a home network will typically have a private IP address.  Attempting to establish a p2p connection to another participant on the ETH2 network using that participant's *private* IP address is not possible, you must use the *public* IP address.  Private IPv4 addresses will always fall into one of the following ranges, as per [RFC1918](https://en.wikipedia.org/wiki/Private_network):
+ - 192.168.0.0 – 192.168.255.255
+ - 172.16.0.0 – 172.31.255.255
+ - 10.0.0.0 – 10.255.255.255
+
+Public IP addresses include all other IP addresses not in the ranges mentioned above, with some exceptions for [Special-Use IPv4 Addresses](https://tools.ietf.org/html/rfc3330).
+
+You can determine your public ip address by visiting (http://v4.ident.me/) or running this commmand:
+```
+curl v4.ident.me
+```
+
+### Port Forwarding
+Participants on home networks will need to configure their router to perform port forwarding so that other ETH2 participants can establish a connection to your beacon-chain node on TCP/13000.  The specific steps required vary based on your router, but can be summarised as follows:
+
+> NOTICE : Participants with nodes on a virtual public cloud (VPC) instance can skip this step.
+
+1) Determine the IP address for your home router
```

Comment:
(Assuming we dont hear differently later this morning) here should note that a quick search will find them many online services to do so

Reply:
done